packages/styles/scss/patterns/cards-with-images/_cards-with-images-group.scss

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../components/link-with-icon/link-with-icon';
+
+@mixin themed-items {
+  .bx--card-link {
+    background-color: $ui-01;
+    border-color: $ui-02;
+    outline: 1px solid $ui-background;
+    &__title {
+      padding: $carbon--spacing-05;
+      color: $text-01;
+    }
+    &__content {
+      padding: $carbon--spacing-05;
+      color: $text-01;
+    }
+    &__footer {
+      svg {
+        fill: $link-01;
+      }
+    }
+  }
+}
+@mixin cards-with-images-group {
+  .#{$prefix}--cards-with-images-group {
+    padding-bottom: $carbon--layout-05;
+    &__title {
+      @include type-style('expressive-heading-02');
```

Comment:
Following the sketch file, we need 64px under the title here:
<img width="709" alt="Screen Shot 2019-12-10 at 4 03 52 PM" src="https://user-images.githubusercontent.com/54281166/70570934-d909c180-1b6a-11ea-8e7e-9a243fbdfbb3.png">

Can you add `padding-bottom: $layout-05` here?

Reply:
Okay